Example/Example/ViewController.m

```diff
@@ -21,11 +24,12 @@ - (void)viewDidLoad {
     // this in a singleton class rather than each time a view loads.
     sdk = [[ATTNSDK alloc] initWithDomain:@"YOUR_ATTENTIVE_DOMAIN" mode:@"production"];
     
+    
     // Register the current user with the Attentive SDK. Replace "APP_USER_ID"
-    // with the current user's ID. You must register a user ID before calling
-    // `trigger` on a Creative.
+    // with the current user's ID.
     // TODO - more info here
-    [sdk identify:@{ IDENTIFIER_TYPE_CLIENT_USER_ID: @"APP_USER_ID", IDENTIFIER_TYPE_PHONE: @"+14156667777"}];
+    _userIdentifiers = @{ IDENTIFIER_TYPE_CLIENT_USER_ID: @"APP_USER_ID", IDENTIFIER_TYPE_PHONE: @"+14156667777"};
```

Comment:
not a blocker for this PR at all, but should we show an example here with all the identifiers & add a comment that says specify as many of these identifiers as you can, but they are all optional?

or should we just show the most important identifiers like phone & email? not sure client user ID is the most important one anymore

Reply:
hmmm. I like showing all of the identifiers. I added to this PR. What do you think?

Reply:
LGTM!